lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.spec.ts

```diff
@@ -57,7 +53,6 @@ import {
     formVariables,
     processVariables,
     mockAllFileSourceWithRenamedFolderVariablePathType,
-    allSourceParamsWithWrongRelativePath,
```

Comment:
The mock imports that you are removing are not used anywhere in the repo. If they are not needed anymore in this or any other repo, consider deleting them.